src/Developer/MultiThreadedImageFilter.h

```diff
@@ -24,10 +24,10 @@ class MultiThreadedImageFilter : public ImageToImageFilter< TImage, TImage >
 
 protected:
   MultiThreadedImageFilter(){}
-  ~MultiThreadedImageFilter(){}
+  ~MultiThreadedImageFilter() override{}
 
-  virtual void ThreadedGenerateData(const OutputImageRegionType &,
-                                    ThreadIdType);
+  void ThreadedGenerateData(const OutputImageRegionType &,
```

Comment:
I guess we should rewrite this example to demonstrate the use of `DynamicThreadedGenerateData`. I created an issue #113 to track this.